Dockerfile

```diff
@@ -2,4 +2,24 @@ FROM openjdk:8-jre
 
 COPY ./modules/application/target/*SNAPSHOT.jar /truthtree/
 
+RUN apt-get update && \
```

Comment:
Don't we have a java + node + git base image from docker repository?

Reply:
I don't see any official docker image with java+node+git. If we use a private one, it will be the same as we do the installation in the Dockerfile I think.